.github/workflows/firefox-ios-ui-tests.yml

```diff
@@ -40,7 +40,8 @@ jobs:
                   -scheme ${{ env.xcodebuild_scheme }} \
                   -target ${{ env.xcodebuild_target }} \
                   -derivedDataPath ~/DerivedData \
-                  -destination 'platform=iOS Simulator,name=${{ env.ios_simulator_default }},OS=${{ env.ios_version }}'
+                  -destination 'platform=iOS Simulator,name=${{ env.ios_simulator_default }},OS=${{ env.ios_version }}' \
+                  COMPILER_INDEX_STORE_ENABLE=NO CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO ARCHS="arm64"
```

Comment:
Without `ARCHS="arm64"`, some seemingly `x64_64` tools has been used in the compilation process. I'm not sure if such tools affected the binary. I'd like to use M1 binaries on the safe side.